### DISCUSSION TITLE
Is there any way to customize the base model?? (Hairstyle, eyes, nose, bodytype etc.)

### DISCUSSION THREAD

**Discussion by Foom-hubs** [mozilla/hubs#3566] [Avatar Questions] [answered]
I have a question about customizing an avatar by utilizing the listed models and changing its skin with Quilt.  I was using only robot models to do this but lately some humanized bodies are also available. I have managed to change their detailed facial expressions and clothes by using a paint tool, and now I can make them look like my friends and guests I'd like to invite to hubs. But those variations of those human bodies are limited. I now need a little roundish face or chubbier body types, also I wanna change cheekbone height, nose shape, as well as hairstyle, which are all fixed and cannot change by changing the skin. Is there any way to modify those basic models? I just wanna make slight changes. I'd appreciate your advice.

**Comment by cvanmeurs** [comment, marked as answer]
Currently no, not outside of editing the model in a 3D application like Blender.  It's something we have been developing and do hope to roll out in the new year.
